python/lsst/analysis/tools/actions/vector/vectorActions.py

```diff
@@ -145,6 +144,18 @@ def __call__(self, data: KeyedData, **kwargs) -> Vector:
         return np.array(cast(Vector, result))
 
 
+class ConstantValue(VectorAction):
```

Comment:
I do need it, yes, but if you think it can be replaced with a generic scalar-to-vector action, that's fine by me.

Reply:
In `KeyedDataActions` we have `KeyedScalars` that does something similar. I think in the spirit of analysis tools where composition is prime focus, maybe a generic `VectorAction` that turns a `ScalarAction` into a vector would be best.

Reply:
Ok, well I'm open to adding that if there's at least one more use case, but since I didn't use the `ConstantAction` and decided to get rid of it entirely, I'd rather wait until there's a more pressing need to introduce that kind of generic scalar-to-vector action.